networkx/algorithms/bridges.py

```diff
@@ -68,20 +70,33 @@ def all_local_bridges(G, first_match=False):
                 G.add_edge(startNode, endNode)
                 if pathLength > 2:
                     # found a local bridge
-                    allBridges.append([startNode, endNode])
-                    allSpans.append(pathLength)
+                    bridgeObjs.append(
+                        {"edge": [startNode, endNode], "span": pathLength})
```

Comment:
Instead of making a list of dictionaries like this, why not have a dictionary of edge -> span? 
`bridges[(start_node, end_node)] = path_length`

The spans aren't meaningful without their corresponding edges, and this lets people who just need the edges to say `for edge in all_local_edges()`